tests/conftest.py

```diff
@@ -115,12 +115,13 @@ def unauthorized_response(mocker, http_error):
     return response
 
 
-@pytest.fixture
-def py42_response(mocker):
+def py42_response(mocker, text):
```

Comment:
I don't think we should be mocking the `Py42Response`.
I think we should just use the already existing `create_mock_response()` method, which mocks the `Response` and passes it into the `Py42Response` class. It is works nicer in my experience.

Reply:
🤦🏼  here I was thinking that I was creating a better way of doing this when one already existed? You're totally right, fortunately the method signature are the same so this should be pretty easy to update en masse

Reply:
Yeah I feel bad as you changed so many! they needed it